cpp/examples/ode_secir_read_graph.cpp

```diff
@@ -123,7 +125,7 @@ int main(int argc, char** argv)
     for (int row = 0; row < twitter_migration_2018.rows(); row++) {
         for (int col = 0; col < twitter_migration_2018.cols(); col++) {
             graph.add_edge(row, col,
-                           Eigen::VectorXd::Constant(8 * (size_t)nb_groups,
+                           Eigen::VectorXd::Constant(10 * (size_t)nb_groups,
```

Comment:
Hi @jubicker , I just stumbled accross this. Even if this had been introduced by me a long time ago, we should remove these hard coded numbers. Please always try to use variables, here InfectionState::Count